pkgs/by-name/co/cosmic-icons/package.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, just
+, pop-icon-theme
+, hicolor-icon-theme
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "cosmic-icons";
+  version = "unstable-2023-08-30";
+
+  src = fetchFromGitHub {
+    owner = "pop-os";
+    repo = pname;
+    rev = "14d8e2048087be1ad444f9b3ebb75885509f72c6";
+    sha256 = "sha256-WbdgHmTn403x95x9wEYL0T9ksbN+YLzEB2yE0UrF9T0=";
+  };
+
+  nativeBuildInputs = [ just ];
+
+  justFlags = [
+    "--set"
+    "prefix"
+    (placeholder "out")
+  ];
+
+  propagatedBuildInputs = [
+    pop-icon-theme
+    hicolor-icon-theme
+  ];
+
+  dontDropIconThemeCache = true;
+
+  meta = with lib; {
+    description = "System76 Cosmic icon theme for Linux";
+    homepage = "https://github.com/pop-os/cosmic-icons";
+    license = with licenses; [
+      cc-by-sa-40
+      gpl3
```

Comment:
Maybe add a comment to clarify if it's "CC-BY-SA-4.0 _and_ GPL3" or "CC-BY-SA-4.0 _or_ GPL3".

Reply:
https://github.com/pop-os/cosmic-icons?tab=readme-ov-file
"This project has mixed licencing. You are free to copy, redistribute and/or modify aspects of this work under the terms of each licence accordingly (unless otherwise specified)." Its or I'm pretty sure.

Reply:
It says quite explicitly that it is different terms for different files:

> The icon assets (any and all source .svg files or rendered .png files) are licenced under the terms of the [Creative Commons Attribution-ShareAlike 4.0 License](https://creativecommons.org/licenses/by-sa/4.0/).
> 
> Included scripts are free software licenced under the terms of the [GNU General Public License, version 3](https://www.gnu.org/licenses/gpl-3.0.txt).

But do the scripts get installed?

Reply:
@alyssais 
We do use the `justfile` which is gpl in order to install the assets, which are `cc-by-sa-40`, does that mean that the installed assets remain `cc-by-sa40` in this case?

Reply:
Yes.  We're not super clear in Nixpkgs in what exactly meta.licenses means.  In this case, since the outputs of the package sound like they're entirely CC BY-SA 4.0, I'd set the license to just that, but if somebody preferred listing both I wouldn't complain.

Reply:
Thank you for the clarification!